about.html

```diff
@@ -5,61 +5,44 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="about.css">
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
 </head>
 <body>
     <header>   
-            <nav> 
-                <img class="logo" src="" alt="">
-                   <ul class="first-li">
-                       <li><a href=""><img src="./images/facebook.png" alt=""></a></li>
-                       <li><a href=""><img src="./images/twitter.png" alt=""></a></li> 
-                       <li><a href="#">english</a></li>
-                       <li><a href="#">my page</a></li>
-                      <li><a href="#">layout</a></li>
-                </ul>
-            </nav>
+        <nav> 
+            <img class="un-k" src="" alt="logo icon">
+               <ul class="first-li">
+                   <li><a href=""><img src="./images/facebook.png" alt="facebook icon"></a></li>
+                   <li><a href=""><img src="./images/twitter.png" alt="twitter icon"></a></li> 
+                   <li><a href="#">english</a></li>
+                   <li><a href="#">my page</a></li>
+                  <li><a href="#">layout</a></li>
+            </ul>
+        </nav>
+
+      <div class="second-item">
+        <img class="logo" src="./images/logo.png" alt="logo icon"> 
+             <ul class="second-li">
+                <li><a class="active" href="index.html">home</a></li>
+                <li><a href="about.html">about</a></li>
+                <li><a href="">join</a></li>
+                <li><a href="">sponsor</a></li
+                <li><a href="#">News</a></li>
+            </ul>
+           
+        </div>
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but still your `about.html` navigation is not looking similar to your `home.html` page. Kindly fix it because right now your design does not look professional at all. I would recommend you also fix other things in your project to make it looks like the original design.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Home Page Design</th>
<th>About Page Design</th>
</tr>
<tr>
<td>

![image](https://github.com/hetrox8/First-Capstone-Project/assets/66476329/547b94b9-8b88-46a3-ba8e-a0ad2a8e3c1d)

</td>
<td>

![image](https://github.com/hetrox8/First-Capstone-Project/assets/66476329/95f49c56-d148-4de6-8130-1f43be019df0)

</td>
</tr>
</table>

Reply:
omg i just can't believe this i literally implement everything each code reviewer ask for. i am having limited chances and this is my last chance else i will be withdrwan from the program.